_posts/2023-10-09-towards_3.0.md

```diff
@@ -0,0 +1,52 @@
+
+
+### A trimmed down core library
+
+As MDAnalysis continues to grow along with the field of molecular simulation, we are aiming to continue to provide a highly functional, easily maintainable package that can be the foundation of a wide variety of downstream tools.
+
+Following the successful introduction of the [MDAKit ecosystem][] (modelled on the SciKit ecosystem, see the [MDAKits whitepaper][]) as part of our [EOSS4][] work, which provides infrastructure for creating FAIR packages that address specific scientific questions in the analysis of molecular simulation, we are aiming to move some more specialized analyses currently present in the MDAnalysis core library to MDAKits. Several analysis modules, including  hole2, encore, path-similarity, helanal and waterdynamics are being assessed to undergo this transition. Indeed some of these packages are already deployed as MDAKits, and are deprecated in the core library.
+
+This change will allow for a reduction in the core dependencies of MDAnalysis and reduce the maintenance burden of specialist analyses in the core library. This will also empower those developing specialist analysis code to rapidly develop and prototype their code in well supported downstream packages. Additionally, core developers will be able to spend more cycles on core competencies, fixing critical issues and planning future improvements, ensuring that MDAnalysis can be easily and effectively maintained for years to come.
+
+### Enhancements
+
+We have identified several areas that we would like to target for future improvement of MDAnalysis across three key themes.
+
+* Improving interoperability
+    - ....
+
+* Support for streaming and cloud storage
+    - ....
+
+* Enhanced parallelism and performance
+    - ...
+
+
+We aim to lay the groundwork for these future directions in our work for MDAnalysis 3.0.
+
+### User facing API changes 
+
+We are also planning to clear up some API inconsistencies scattered across the library that require an API break, including some instances where coordinate readers behave differently relative to each other, renaming some arguments to clearer names and addressing other miscellanea. Additionally, we are aiming to provide a more consistent and usable low-level Cython API surface that compiled extensions can hook into. We are also investigating the fesability of improving consistency of precision handling in various parts of the MDAnalysis core library, as the use of double and mixed precision has been identified as a key performance bottleneck.
+
+
+### Improving org structure and formalising roles and responsibilities. 
+
+We have identified that assigning direct roles and responsibilities across our coredevs is a change that will be beneficial for the health of our community going forward.
+To this end we are workshopping a more formal org structure that will make our internal processes more transparent and allow more functional redundancy in key roles. Watch this space!
```

Comment:
Thanks for including this, I guess it's not part of the code and as part of 3.0 but IMO it's definitely something we identified as an organisational priority for MDA as a whole!